src/analysis/seminorm.lean

```diff
@@ -762,9 +764,7 @@ noncomputable instance : lattice (seminorm 𝕜 E) :=
     simp only [sub_self, seminorm.zero, zero_add, sub_zero], end,
   le_inf := λ a b c hab hac x, begin
     apply le_cinfi, intro u,
-    have s : x = u + (x - u), { abel },
-    { conv_lhs { rw s },
-      apply le_trans (a.triangle u (x-u)) (add_le_add (hab u) (hac (x-u))) },
+    exact le_trans (a.le_insert' x u) (add_le_add (hab u) (hac (x-u))),
```

Comment:
We tend to use underscores to indicate that Lean correctly guesses the argument and to reduce visual overload.
```suggestion
    exact le_trans (a.le_insert' _ _) (add_le_add (hab _) (hac _)),
```